web/docs/contributing.md

```diff
@@ -4,7 +4,7 @@ sidebar_label: Contributing
 slug: /contributing
 ---
 
-import DiscordLink from '../blog/components/DiscordLink';
+import DiscordLink from '@site/blog/components/DiscordLink';
```

Comment:
Relative links don't work with versioned docs, so it is recommended to use `@site`, which we use in most places anyway, but we had a couple of places where we didn't use it, which I fixed.